modules/test/src/main/java/io/fluo/recipes/test/FluoITHelper.java

```diff
@@ -0,0 +1,435 @@
+  private AtomicInteger fluoTableCounter = new AtomicInteger(1);
+  private AtomicBoolean fluoRunning = new AtomicBoolean(false);
+
+  public FluoITHelper(Path baseDir) {
+    this.baseDir = baseDir;
+    Objects.requireNonNull(baseDir);
+    try {
+      FileUtils.deleteDirectory(baseDir.toFile());
+      MiniAccumuloConfig cfg = new MiniAccumuloConfig(baseDir.toFile(), ACCUMULO_PASSWORD);
+      cluster = new MiniAccumuloCluster(cfg);
+      cluster.start();
+    } catch (IOException | InterruptedException e) {
+      throw new IllegalStateException(e);
+    }
+    updateFluoConfig();
+  }
+
+  /**
+   * Prints list of RowColumnValue objects
+   *
+   * @param rcvList RowColumnValue list
+   */
+  public static void printRowColumnValues(List<RowColumnValue> rcvList) {
+    System.out.println("== RDD start ==");
+    rcvList.forEach(rcv -> System.out.println("rc " + Hex.encNonAscii(rcv, " ")));
+    System.out.println("== RDD end ==");
+  }
+
+  /**
+   * Prints default Fluo table
+   */
+  public void printFluoTable() {
+    try (FluoClient fc = FluoFactory.newClient(miniFluo.getClientConfiguration())) {
+      printFluoTable(fc);
+    }
+  }
+
+  /**
+   * Prints Fluo table accessible using provided client
+   *
+   * @param client Fluo client to table
+   */
+  public static void printFluoTable(FluoClient client) {
+    try (Snapshot s = client.newSnapshot()) {
+      RowIterator iter = s.get(new ScannerConfiguration());
+
+      System.out.println("== fluo start ==");
+      while (iter.hasNext()) {
+        Map.Entry<Bytes, ColumnIterator> rowEntry = iter.next();
+        ColumnIterator citer = rowEntry.getValue();
+        while (citer.hasNext()) {
+          Map.Entry<Column, Bytes> colEntry = citer.next();
+
+          StringBuilder sb = new StringBuilder();
+          Hex.encNonAscii(sb, rowEntry.getKey());
+          sb.append(" ");
+          Hex.encNonAscii(sb, colEntry.getKey(), " ");
+          sb.append("\t");
+          Hex.encNonAscii(sb, colEntry.getValue());
+
+          System.out.println(sb.toString());
+        }
+      }
+      System.out.println("=== fluo end ===");
+    }
+  }
+
+  /**
+   * Verifies that the actual data in the default Fluo instance of FluoITHelper matches expected
+   * data
+   *
+   * @param expected RowColumnValue list containing expected data
+   * @return True if actual data matches expected data
+   */
+  public boolean verifyFluoTable(List<RowColumnValue> expected) {
+    try (FluoClient fc = FluoFactory.newClient(miniFluo.getClientConfiguration())) {
+      return verifyFluoTable(fc, expected);
+    }
+  }
+
+  /**
+   * Verifies that the actual data in provided Fluo instance matches expected data
+   *
+   * @param client Fluo client to instance with actual data
+   * @param expected RowColumnValue list containing expected data
+   * @return True if actual data matches expected data
+   */
+  public static boolean verifyFluoTable(FluoClient client, List<RowColumnValue> expected) {
```

Comment:
It seems like the intention is that expected should be sorted.  Would `SortedSet<RowColumnValue>` be a better type to use than `List<RowColumnValue>` ?